docs/adrs/001-start-using-adrs.md

```diff
@@ -0,0 +1,42 @@
+---
+status: decided
+date: 17-05-2024
+deciders: Kamil Kołodziej
+---
+# Start using ADRs to document any major (non)architectural decisions for the project
+
+## Context and Problem Statement
+
+Right now, we don't have any place where we keep decisions about the project.
+We would like to keep a log of any major decisions made which affect the application.
+
+## Decision Drivers
+
+* One place to keep all the decisions made
+* Easy to read and write
+* Close to the code
+
+## Considered Options
+
+* ADRs inside the repo
+* Google Docs
+* Confluence
```

Comment:
**question**: Shouldn't `Google Docs` and `Confluence` be at least shortly described somewhere in this ADR?

Reply:
Well maybe in something more serious (like the one below) I'm all for describing the options but here I'm just not feeling the need. One could deduct from the Decision outcome why we don't choose the other two.